pages/admin/auth/login.tsx

```diff
@@ -96,6 +99,16 @@ const Login = ({ csrfToken }: InferGetServerSidePropsType<typeof getServerSidePr
                 </div>
               </div>
             </form>
+            <SSOLogin
+              buttonText={t('login_with_sso')}
+              ssoIdentifier={`tenant=${tenant}&product=${product}`}
+              onSubmit={onSSOSubmit}
+              classNames={{
+                container: 'mt-2',
```

Comment:
Would it be better if we change the "Login with SSO" button as secondary button? Also both buttons have different border radius. We should make them consistent.

Reply:
I feel both the buttons have equal prominence. Maybe both of them can be set to the `btn-outline` variant. 

<img width="432" alt="Screenshot 2023-01-06 at 2 57 30 PM" src="https://user-images.githubusercontent.com/3107922/210971914-7c02be94-1e0b-4912-a90f-fad2427eacf8.png">


@deepakprabhakara Thoughts ?